substrate/bin/node-template/pallets/template/src/tests.rs

```diff
@@ -1,4 +1,4 @@
-use crate::{mock::*, Error, Event};
+use crate::{mock::*, Error, Event, Something};
```

Comment:
Can you update https://github.com/substrate-developer-hub/substrate-node-template as well? see https://github.com/paritytech/polkadot-sdk/issues/3155